templates/volunteer/base.html

```diff
@@ -5,8 +5,9 @@
         <meta http-equiv="X-UA-Compatible" content="IE=edge,chrome=1">
         <meta name="viewport" content="width=device-width, initial-scale=1.0">
         <title>{% block title %}{% endblock %}{% if self.title() %} - {% endif %}EMF Volunteering</title>
-        <link rel="stylesheet" href="https://maxcdn.bootstrapcdn.com/bootstrap/3.3.7/css/bootstrap.min.css" integrity="sha384-BVYiiSIFeK1dGmJRAkycuHAHRg32OmUcww7on3RYdg4Va+PmSTsz/K68vbdEjh4u" crossorigin="anonymous">
-        <link rel="stylesheet" href="{{ static_url_for('static', filename='css/volunteer.css') }}">
+        {% block css %}
+           <link rel="stylesheet" href="{{ static_url_for('static', filename="css/main.css") }}">
```

Comment:
This looks like `css/volunteer.css` was missed in translation, however the file no longer exists.